docs/configuration/parsing-methods.rst

```diff
@@ -0,0 +1,57 @@
+.. _parsing-methods:
+
+Parsing Methods
+===============
+
```

Comment:
Should we give a high-level example of how to import LoadMethod and how to use it within a `DbtDag` or `DbtTaskGroup`?